packages/ckeditor5-utils/tests/dom/position.js

```diff
@@ -688,8 +688,9 @@ describe( 'getOptimalPosition()', () => {
 
 function assertPosition( options, expected ) {
```

Comment:
I'm missing some tests that will verify that:

*   The output of `getOptimalPosition` is an instance of `Position`
*   That `config` from the positioning function is correctly passed-through `getOptimalPosition` to the `Position` (depending whether positioning function defined it or not - see my earlier comment about the manual test).

Reply:
Done.